### ISSUE TITLE
Very disappointed

### ISSUE THREAD

**Issue by tfmorris** [OpenRefine/OpenRefine#359]
_Original author: jmangr...@gmail.com (April 12, 2011 18:29:29)_

I am a data analyst (among other things) who routinely works with large datasets sometimes requiring normalization.  My normal tools are MS Excel and Access.  So, when I heard about Refine I couldn't wait to try it.

I hate to criticize others' work so negatively, but what a disappointment!  Here's a partial list of the issues I encountered.

. Refine could not &quot;facet&quot; a dataset of about 11K records on a column with a little more than 8K unique values (&quot;too many values to display&quot;).  That's a fairly small dataset in the kind of work I do, and not a lot of unique values either.

. Could not filter on a simple NOT condition.  Apparently Refine filtering supports simple absolute string matching and for all other purposes uses &quot;regular expressions&quot; and/or other forms of code.  If I've misunderstood Refine's capabilities I apologize, but would like to point out that this is the extent of my understanding after scouring the available user documentation.

. Refine ran for 5 minutes faceting my 11K record dataset on a simple integer numeric field and never finished.  I finally shut Refine down as I had no idea how close to done it was.  A progress indicator would help a lot.

. No way to jump to a random ordinal record in the dataset.  Refine only lets you navigate to the beginning, end, next or previous record.  This is useless for datasets where one often needs to jump to the middle somewhere, especially in large datasets.

. Export does not export flags and stars!  I had to do two exports, one of only flagged records and the other of all records and rejoin in Access to work around this limitation.

. Export does not export in sorted sequence.  If you sort the dataset in Refine and then export it the sort sequence is lost in the output.

. And last but not least, please produce a real user's guide that gives explanations of all its main features and examples of their use as well as a proper table of contents and an index.  A function context help system (a la Microsoft, sorry to say) would also work.  Videos and scenarios such as the user documentation Web site provides are nice, but they're useless when one is trying to figure out how to do something different from the examples.

In short, I'm very disappointed because Refine's visualization features would be very useful in my work, but they are not available to me because of the product's severe limitations.  I'm sure it's a great product for users with really small datasets to manipulate, but that pretty much leaves me out.

_Original issue: http://code.google.com/p/google-refine/issues/detail?id=359_
